### ISSUE TITLE
Is this library even being managed any more?

### ISSUE THREAD

**Issue by oasjohn** [daybrush/moveable#1128]
If not, should I just move on and write my own or is someone still handling issues?

**Comment by jogibear9988** [comment]
If you need to work on something for yourself my https://github.com/jogibear9988/getBoxQuadsPolyfill could help.
I use this also in my designer: https://github.com/node-projects/web-component-designer

**Comment by oasjohn** [comment]
Thanks, but it would be nice to get a response from anyone managing this repo. Seems like it's abandoned.

**Comment by ajiho** [comment]
Obviously, it has been abandoned. Perhaps you should implement the logic yourself so that you can better control it if there are any issues

**Comment by alfredomariamilano** [comment]
> Obviously, it has been abandoned. Perhaps you should implement the logic yourself so that you can better control it if there are any issues

@ajiho I understand your response, but dude, the different interconnected projects that make up `moveable` are massive and they're a huge undertaking, so it's not as easy as writing it yourself.

**Comment by VMBindraban** [comment]
@daybrush Can you clarify if this project is abandoned?

**Comment by li1164267803** [comment]
To answer the question in the title: as of today, no — but I've opened #1156 offering to help, and emailed @daybrush directly. No response yet.

@alfredomariamilano's point is the one I'd underline. These projects are interconnected and large — moveable, selecto, gesto and keycon all stalled within two months of each other in mid-2024 — which is exactly why "just write it yourself" isn't a real answer for most teams.

I've been running react-moveable in production for a year and have built persistent group support on top of it: group/ungroup with rotation-aware coordinate conversion, recursive group resize dispatched by child type, and rotation snapping. I'd much rather contribute that upstream than maintain it privately.

If you're carrying local patches of your own, #1156 is a good place to say so.

**Comment by li1164267803** [comment]
To answer the question in the title: as of today, no — but I've opened #1156 offering to help, and emailed @daybrush directly. No response yet.

@alfredomariamilano's point is the one I'd underline. These projects are interconnected and large, and they went quiet together — the last commit on `master` is 2023-12-03 for moveable and selecto, 2023-12-01 for gesto, and 2024-07-29 for keycon. That is exactly why "just write it yourself" isn't a real answer for most teams.

I've been running `react-moveable` in production for a year and built the layer Moveable deliberately leaves out — persistent groups: group/ungroup with rotation-aware conversion between a child's absolute position and its position relative to the group center, and recursive group resize dispatched by child type. I'd much rather contribute that upstream than maintain it privately.

If you're carrying local patches of your own, #1156 is a good place to say so.

**Comment by jogibear9988** [comment]
If someone is workin on this, I can now say the getBoxQuads API has now landed in Chrome (still behind a flag, but will be opened). So the use of https://github.com/jogibear9988/getBoxQuadsPolyfill and getBoxQuads instead of calculating points manually would be a good idea!

**Comment by li1164267803** [comment]
> If someone is workin on this, I can now say the getBoxQuads API has now landed in Chrome (still behind a flag, but will be opened). So the use of https://github.com/jogibear9988/getBoxQuadsPolyfill and getBoxQuads instead of calculating points manually would be a good idea!

I really hope these features can all be merged together. Right now, I can only hope that I can have the corresponding permissions unified in the upstream.

**Comment by jogibear9988** [comment]
Create a fork and maintain it. This always works

**Comment by li1164267803** [comment]
> Create a fork and maintain it. This always works

If there's no response, you'll have to fork and maintain it yourself. A drawback of this is that others might not know what you've done and will continue using the old version.
